crates/noirc_evaluator/src/ssa/context.rs

```diff
@@ -47,6 +48,16 @@ impl<'a> SsaContext<'a> {
         pc
     }
 
+    pub fn one(&self) -> NodeId {
+        self.find_const_with_type(&BigUint::one(), node::ObjectType::Unsigned(1))
+            .unwrap()
```

Comment:
Are the 0 and 1 consts guaranteed to always be present?

Reply:
yes, they are created in the constructor

Reply:
I removed them because they are not used anymore